app/models/katello/flatpak_remote.rb

```diff
@@ -0,0 +1,13 @@
+module Katello
+  class FlatpakRemote < Katello::Model
+    has_many :katello_flatpak_remote_repositories, dependent: :destroy, class_name: 'Katello::FlatpakRemoteRepository'
+    has_many :katello_flatpak_remote_repository_manifests, through: :katello_flatpak_remote_repositories, source: :katello_flatpak_remote_repository_manifests
```

Comment:
Any reason why the `has_many` relationships here are prefixed with `katello`? I think just `flatpak_remote_repository_manifests` could work.

Reply:
Updated to shorter names for the relationships..Let me know if those make sense.